src/interfaces/IMetaMorpho.sol

```diff
@@ -70,36 +70,49 @@ interface IMetaMorphoBase {
     function withdrawQueueLength() external view returns (uint256);
 
     /// @notice Stores the total assets managed by this vault when the fee was last accrued.
-    /// @dev May be a little off `totalAssets()` after each interaction, due to some roundings.
+    /// @dev May be greater than `totalAssets()` due to removal of markets with non-zero supply or socialized bad debt.
```

Comment:
Consider that this is only temporary until a "sync" operation is triggered

Reply:
you mean we should make it clearer?

Reply:
yes

Reply:
I think this is what the line below is mentioning but maybe it's not clear enough